### ISSUE TITLE
Gamepad support?

### ISSUE THREAD

**Issue by chstan** [CrimsonZen/opbox#4]

**Comment by CrimsonZen** [comment]
I was thinking about this - it would be cute to get it working with a gamecube or wii u pro controller.

**Comment by chstan** [comment]
Yeah that would be awesome. There's the Gamepad API but I'm not sure what it's like to actually work with.

**Comment by chstan** [comment]
If we went this route in the future there would probably have to be some configurability around control layout, at least to the point of supporting different controllers.
